### DISCUSSION TITLE
Chocolatey adaptation?

### DISCUSSION THREAD

**Discussion by DailenG** [Romanitho/Winget-AutoUpdate#348] [General]
There is a few similar projects that do this but I'm particularly fond of Winget-AutoUpdate and WiGui. But how difficult do you believe an adaptation to include Chocolatey support be? Maybe not even as a new feature perhaps as a fork.

Clients I work with do not utilize Intune so Chocolatey is easier to create repositories for internal installations.

**Comment by Romanitho** [comment]
Hi,
I was a chocolatey user in the past. But as I'm an Intune integrator, I spend my time to Winget and this project :)
Today, I guess it is not that hard to adapt this project to choco. But i will not have time for this as my free time is limited currently.
